src/en/weebdex/src/eu/kanade/tachiyomi/extension/en/weebdex/WeebDex.kt

```diff
@@ -0,0 +1,271 @@
+package eu.kanade.tachiyomi.extension.en.weebdex
+
+import eu.kanade.tachiyomi.extension.en.weebdex.dto.ChapterDto
+import eu.kanade.tachiyomi.extension.en.weebdex.dto.ChapterListDto
+import eu.kanade.tachiyomi.extension.en.weebdex.dto.MangaDto
+import eu.kanade.tachiyomi.extension.en.weebdex.dto.MangaListDto
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import okhttp3.Headers
+import okhttp3.HttpUrl.Companion.toHttpUrlOrNull
+import okhttp3.Request
+import okhttp3.Response
+import java.text.SimpleDateFormat
+import java.util.Locale
+import java.util.TimeZone
+
+class WeebDex : HttpSource() {
+    override val name = "WeebDex"
+    override val baseUrl = "https://weebdex.org"
+    override val lang = "en"
+    override val supportsLatest = true
+
+    override val client = network.cloudflareClient.newBuilder()
+        .rateLimit(WeebDexConstants.rateLimitUsed)
+        .build()
+
+    override fun headersBuilder(): Headers.Builder = Headers.Builder()
+        .add("User-Agent", "Tachiyomi " + System.getProperty("http.agent"))
```

Comment:
Use `super.headersBuilder()` to preserve default headers from the app.
Idk if should identify the app making a custom user agent
```suggestion
    override fun headersBuilder(): Headers.Builder = super.headersBuilder()
        .add("User-Agent", "Tachiyomi " + System.getProperty("http.agent"))
```

Reply:
While probably not needed, i saw mangadex doing this, so i kept it as is. Shouldn't cause any issues.

Reply:
don't set custom useragent, if they ask for it only then we should send identifying useragent.